packages/gatsby-telemetry/src/event-storage.ts

```diff
@@ -1,34 +1,44 @@
-const os = require(`os`)
-const path = require(`path`)
-const Store = require(`./store`)
-const fetch = require(`node-fetch`)
-const Configstore = require(`configstore`)
-const { ensureDirSync } = require(`fs-extra`)
-
+import os from "os"
+import path from "path"
+import Configstore from "configstore"
+import createFetch from "@turist/fetch"
+import { Store } from "./store"
+import { ensureDirSync } from "fs-extra"
 import { isTruthy } from "./is-truthy"
 
+const fetch = createFetch()
+
 /* The events data collection is a spooled process that
  * buffers events to a local fs based buffer
  * which then is asynchronously flushed to the server.
- * This both increases the fault tolerancy and allows collection
+ * This both increases the fault tolerance and allows collection
  * to continue even when working offline.
  */
 module.exports = class EventStorage {
   analyticsApi =
     process.env.GATSBY_TELEMETRY_API || `https://analytics.gatsbyjs.com/events`
+  // eslint-disable-line @typescript-eslint/no-explicit-any
+  config: Configstore | Record<string, any>
```

Comment:
Why could this be a Record?

Reply:
if $home is not writable workaround is in use

Reply:
But you're casting that to a Configstore. Better though to use that as a separate InMemoryConfigStore class

Reply:
```
[tsserver 2322] [E] Type '(key: string, value: any) => void' is not assignableto type '{ (key: string, val: any): void; (values: any): void; }'
```

Reply:
lol, extracting it so that I don't have to figure this out :D